rpc/src/event.rs

```diff
@@ -46,17 +47,17 @@ pub enum EventData {
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct TxInfo {
     pub height: String,
-    pub index: i64,
+    pub index: Option<i64>,
```

Comment:
Weird that this could be empty I'd expect it to just be 0, I guess it's an `omitempty` on the Go side - https://github.com/tendermint/tendermint/blob/master/abci/types/types.pb.go#L2764